src/Adapter/MSTest.TestAdapter/Execution/LogMessageListener.cs

```diff
@@ -169,5 +169,7 @@ public void Dispose()
 
             s_listenerCount--;
         }
+
+        GC.SuppressFinalize(this);
```

Comment:
I doubt it will change anything as the class doesn't seem to declare a finalizer at all, but I'm not opposed if this was some warning resulting from analyzers update imposed by the SDK update.

Reply:
Yes, this is just a new analyzer complaining.